libmscore/measurebase.cpp

```diff
@@ -307,8 +307,11 @@ void MeasureBase::layout()
 
 void MeasureBase::triggerLayout() const
       {
-      if (prev() || next()) // avoid triggering layout before getting added to a score
-            score()->setLayout(tick(), -1, this);
+      // for measurebases within other measurebases (e.g., hbox within vbox), use parent
+      const MeasureBase* mb = parent() && parent()->isMeasureBase() ? toMeasureBase(parent()) : this;
```

Comment:
Is it possible to have boxes nested to more than one level? When trying to figure this out I only managed to crash MuseScore (see [this issue](https://musescore.org/en/node/307929)) but if there isn't any limitation on a number of nested boxes then it might be better to find a topmost `MeasureBase` parent here with a loop.

Reply:
It shouldn't be possible to have done this at all as far as I know, but OK, yeah, better safe than sorry.  So I should probably factor this loop out into its own function for reuse.